ui/DesignSystem/Component/Timeline/TimelineItem.svelte

```diff
@@ -0,0 +1,132 @@
+<script>
```

Comment:
Naming this `Item.svelte` would make more sense as it lives under `Timeline` already.